### PR TITLE
Defaults tab design update

### PR DESCRIPTION
**Pull request by sagarpatelr**

### CODE REVIEW

File: src/resources/scss/admin/_fields.scss [hunk 1]
```diff
@@ -235,16 +232,20 @@
...
-		border: 1px solid #bbb;
+		background: var( --pngx-white ) url('data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%2211%22%20height%3D%2211%22%20viewBox%3D%220%200%2011%2011%22%3E%3Cpath%20d%3D%22M4.33%208.5L0%201L8.66%201z%22%20fill%3D%22%23666%22%2F%3E%3C%2Fsvg%3E') right 10px center no-repeat;
+		border: 1px solid var( --pngx-grey-5 );
 		/* <svg xmlns="http://www.w3.org/2000/svg" width="11" height="11" viewBox="0 0 11 11"><path d="M4.33 8.5L0 1L8.66 1z" fill="#666"/></svg> */
-		border-radius: 2px;
+		border-radius: 0.5rem;
 
 		select {
-			font-size: 16px;
+			font-size: 0.875rem;
```
Comment (jesseeproductions) [inline src/resources/scss/admin/_fields.scss]: @sagarpatelr  In this PR can you please go through and change to use the rem-calc() function. 

Let me know if it is not working for you, but I tested and was able to get it to work. 

Thanks

Comment (sagarpatelr) [reply src/resources/scss/admin/_fields.scss]: Do we need to change here only or need to change everywhere where I've used "rem"?

Comment (jesseeproductions) [reply src/resources/scss/admin/_fields.scss]: Please change it everywhere that you changed.

Comment (sagarpatelr) [reply src/resources/scss/admin/_fields.scss]: Changes has been updated and committed as well.

**Pull request merged**